### ISSUE TITLE
Support Heralding honeypot

### ISSUE THREAD

**Issue by t3chn0m4g3** [vorband/ewsposter#3]
Include a config section to support heralding honeypot.

Logfile example `/data/heralding/log/auth.csv`:
```
timestamp,auth_id,session_id,source_ip,source_port,destination_ip,destination_port,protocol,username,password
2018-04-24 14:12:29.226769,28bf7135-7007-496d-b284-6112b5d69176,d27bc0b7-3465-43a7-b5d7-65cf310e363e,172.21.0.1,43288,87.150.238.182,22,ssh,root,1337
2018-04-24 14:12:34.170653,f093252c-2888-45cc-bc88-bdb6a18d8863,d27bc0b7-3465-43a7-b5d7-65cf310e363e,172.21.0.1,43288,87.150.238.182,22,ssh,root,iamwithstupid
2018-04-24 14:12:38.842357,e5155baa-ba98-4b8a-add1-15db2ddf048c,d27bc0b7-3465-43a7-b5d7-65cf310e363e,172.21.0.1,43288,87.150.238.182,22,ssh,root,passw0rd
2018-04-24 14:12:54.785686,f6df0b8e-9292-4521-a86c-f9f458283193,da588bf3-15c1-42a2-8d5d-b08533fc2e26,172.21.0.1,50800,87.150.238.182,21,ftp,john,doe
```

**Comment by vorband** [comment]
is the first line of the log always _timestamp,auth_id,session_id,source_ip,source_port,destination_ip,destination_port,protocol,username,password_ ?

**Comment by t3chn0m4g3** [comment]
Yes, the line seems to be written only once heralding is starting and the log file is still empty. Repeated restarting of the honeypot and invoking new log entries will not reproduce the corresponding line as far as I can see (100+ entries and no repetition).

**Comment by vorband** [comment]
implemented in ace224cbe04499ba63f3a1b2ae36b507da668056

Please test and close if working

**Comment by t3chn0m4g3** [comment]
Tested a few services and LGTM:
```
 => Starting HERALDING Modul.
 => Send Limit is set to : 500. Adapting to limit!
    -> 3 EWS alert records send ...
```
Thank you! 😄
